### ISSUE TITLE
[Driver.py] TODO

### ISSUE THREAD

**Issue by ahnjaeshin** [yuyupopo/DeepPano#3]
# TODO
- [x] new metric (DICE)
- [x] Weight initialization (flag)
- [x] Log 버그 고치기
- [x] close writer when get keyboard exception http://effbot.org/zone/stupid-exceptions-keyboardinterrupt.htm
==================== (till midterm)
- [ ] pytorch 0.4 버전으로 올리기
- [ ] tensor board 새로 유저로 설치
- [ ] Name convention
- [ ] log file manage
- [ ] json config
- [ ] refactoring train.py
- [ ] ready for pretrain / negative sampling
- [ ] hyper parameter search
- [ ] colab 사용
- [ ] automatic report generation
- [ ] log (text) 정상화
- [ ] gpu 사용량 체크
===================== optional

> to be added
